### DISCUSSION TITLE
Snapmaker Start Code Set Temps

### DISCUSSION THREAD

**Discussion by jorgeszabo** [Kaouthia/Snapmaker-2#6] [General]
Hi,
Working through the setup of the Snapmaker in Cura I noticed that the start code for nozzle and bed temperature is currently defined as the final printing temperature and not the initial layer temperature. In order to be able to use the settings for the initial temperature the start code for the temperature commands can be modified to:

M104 S{material_print_temperature_layer_0} ;Set Hotend Temperature
M140 S{material_bed_temperature_layer_0} ;Set Bed Temperature

M109 S{material_print_temperature_layer_0} ;Wait for Hotend Temperature
M190 S{material_bed_temperature_layer_0} ;Wait for Bed Temperature

**Comment by Kaouthia** [comment]
I'm not sure if you're looking at an older version of the file, but they've been set to do exactly what you suggest since January :)

https://github.com/Kaouthia/Snapmaker-2/blob/main/Cura%20Profiles/start-g-code.txt

**Reply by jorgeszabo** [reply]
Thanks! It does seem that I overlooked it.
